jwst/charge_migration/charge_migration.py

```diff
@@ -13,23 +11,26 @@
 
 CHLO_DNU = CHLO + DNU
 
-__all__ = ["charge_migration", "flag_pixels"]
```

Comment:
I doubt it, but have you made sure `flag_pixels` is not used elsewhere, since it's in the `__all__`?